cpp/src/utilities/logger.cpp

```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
+#include "spdlog/sinks/stdout_sinks.h"
```

Comment:
This should match with its companion at line 22.
https://github.com/rapidsai/cudf/pull/15063/files#diff-906fc8cdd3162b1f4778c3e85d9e72505ff171ccaaaab5c4a8f704f2cef7637eR22

Reply:
Fixed, thx.